### ISSUE TITLE
Undefined method `include?`

### ISSUE THREAD

**Issue by iovuio** [asciidoctor/asciidoctor-intellij-plugin#239]
**Steps to reproduce and context**
I could not find the exact content that triggers this, however I am doing live edit of a lot of AsciiDoc files, so the plugin encounters content in unstable states (i.e. unterminated tables). The exception thrown several times over the course of a couple of hours. I can see in the event log:

`Error rendering asciidoctor`

Please see the stack trace at the end of this text.

Thank you for your great plugin.

**AsciiDoc Plugin Version** : v0.24.1

**IDE Name and Version**: IntelliJ IDEA 2018.3.5 (Ultimate Edition)
Build #IU-183.5912.21, built on February 26, 2019
JRE: 1.8.0_152-release-1343-b28 x86_64
JVM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
macOS 10.14.3

**Stack trace** 

```
Error rendering preview: org.jruby.exceptions.NoMethodError: (NoMethodError) undefined method `include?' for nil:NilClass
Did you mean?  include_class

org.asciidoctor.internal.AsciidoctorCoreException: org.jruby.exceptions.NoMethodError: (NoMethodError) undefined method `include?' for nil:NilClass
Did you mean?  include_class
	at org.asciidoctor.internal.JRubyAsciidoctor.convert(JRubyAsciidoctor.java:421)
	at org.asciidoctor.internal.JRubyAsciidoctor.convert(JRubyAsciidoctor.java:384)
	at org.asciidoctor.internal.JRubyAsciidoctor.render(JRubyAsciidoctor.java:210)
	at org.asciidoc.intellij.AsciiDoc.render(AsciiDoc.java:204)
	at org.asciidoc.intellij.editor.AsciiDocPreviewEditor$2.run(AsciiDocPreviewEditor.java:159)
	at org.asciidoc.intellij.editor.LazyApplicationPoolExecutor$1.run(LazyApplicationPoolExecutor.java:59)
	at com.intellij.openapi.application.impl.ApplicationImpl$1.run(ApplicationImpl.java:314)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
Caused by: org.jruby.exceptions.NoMethodError: (NoMethodError) undefined method `include?' for nil:NilClass
Did you mean?  include_class
	at uri_3a_classloader_3a_.gems.asciidoctor_minus_1_dot_5_dot_8.lib.asciidoctor.converter.html5.stem(uri:classloader:/gems/asciidoctor-1.5.8/lib/asciidoctor/converter/html5.rb:654)
	at uri_3a_classloader_3a_.gems.asciidoctor_minus_1_dot_5_dot_8.lib.asciidoctor.converter.base.convert(uri:classloader:/gems/asciidoctor-1.5.8/lib/asciidoctor/converter/base.rb:38)
	at uri_3a_classloader_3a_.gems.asciidoctor_minus_1_dot_5_dot_8.lib.asciidoctor.abstract_block.convert(uri:classloader:/gems/asciidoctor-1.5.8/lib/asciidoctor/abstract_block.rb:75)
	at uri_3a_classloader_3a_.gems.asciidoctor_minus_1_dot_5_dot_8.lib.asciidoctor.abstract_block.content(uri:classloader:/gems/asciidoctor-1.5.8/lib/asciidoctor/abstract_block.rb:84)
	at org.jruby.RubyArray.map(org/jruby/RubyArray.java:2577)
	at uri_3a_classloader_3a_.gems.asciidoctor_minus_1_dot_5_dot_8.lib.asciidoctor.abstract_block.content(uri:classloader:/gems/asciidoctor-1.5.8/lib/asciidoctor/abstract_block.rb:84)
	at uri_3a_classloader_3a_.gems.asciidoctor_minus_1_dot_5_dot_8.lib.asciidoctor.converter.html5.section(uri:classloader:/gems/asciidoctor-1.5.8/lib/asciidoctor/converter/html5.rb:369)
	at uri_3a_classloader_3a_.gems.asciidoctor_minus_1_dot_5_dot_8.lib.asciidoctor.converter.base.convert(uri:classloader:/gems/asciidoctor-1.5.8/lib/asciidoctor/converter/base.rb:38)
	at uri_3a_classloader_3a_.gems.asciidoctor_minus_1_dot_5_dot_8.lib.asciidoctor.abstract_block.convert(uri:classloader:/gems/asciidoctor-1.5.8/lib/asciidoctor/abstract_block.rb:75)
	at uri_3a_classloader_3a_.gems.asciidoctor_minus_1_dot_5_dot_8.lib.asciidoctor.abstract_block.content(uri:classloader:/gems/asciidoctor-1.5.8/lib/asciidoctor/abstract_block.rb:84)
	at org.jruby.RubyArray.map(org/jruby/RubyArray.java:2577)
	at uri_3a_classloader_3a_.gems.asciidoctor_minus_1_dot_5_dot_8.lib.asciidoctor.abstract_block.content(uri:classloader:/gems/asciidoctor-1.5.8/lib/asciidoctor/abstract_block.rb:84)
	at uri_3a_classloader_3a_.gems.asciidoctor_minus_1_dot_5_dot_8.lib.asciidoctor.converter.html5.section(uri:classloader:/gems/asciidoctor-1.5.8/lib/asciidoctor/converter/html5.rb:368)
	at uri_3a_classloader_3a_.gems.asciidoctor_minus_1_dot_5_dot_8.lib.asciidoctor.converter.base.convert(uri:classloader:/gems/asciidoctor-1.5.8/lib/asciidoctor/converter/base.rb:38)
	at uri_3a_classloader_3a_.gems.asciidoctor_minus_1_dot_5_dot_8.lib.asciidoctor.abstract_block.convert(uri:classloader:/gems/asciidoctor-1.5.8/lib/asciidoctor/abstract_block.rb:75)
	at uri_3a_classloader_3a_.gems.asciidoctor_minus_1_dot_5_dot_8.lib.asciidoctor.abstract_block.content(uri:classloader:/gems/asciidoctor-1.5.8/lib/asciidoctor/abstract_block.rb:84)
	at org.jruby.RubyArray.map(org/jruby/RubyArray.java:2577)
	at uri_3a_classloader_3a_.gems.asciidoctor_minus_1_dot_5_dot_8.lib.asciidoctor.abstract_block.content(uri:classloader:/gems/asciidoctor-1.5.8/lib/asciidoctor/abstract_block.rb:84)
	at uri_3a_classloader_3a_.gems.asciidoctor_minus_1_dot_5_dot_8.lib.asciidoctor.document.content(uri:classloader:/gems/asciidoctor-1.5.8/lib/asciidoctor/document.rb:1261)
	at uri_3a_classloader_3a_.gems.asciidoctor_minus_1_dot_5_dot_8.lib.asciidoctor.converter.html5.embedded(uri:classloader:/gems/asciidoctor-1.5.8/lib/asciidoctor/converter/html5.rb:270)
	at uri_3a_classloader_3a_.gems.asciidoctor_minus_1_dot_5_dot_8.lib.asciidoctor.converter.base.convert(uri:classloader:/gems/asciidoctor-1.5.8/lib/asciidoctor/converter/base.rb:38)
	at uri_3a_classloader_3a_.gems.asciidoctor_minus_1_dot_5_dot_8.lib.asciidoctor.document.convert(uri:classloader:/gems/asciidoctor-1.5.8/lib/asciidoctor/document.rb:1190)
	at uri_3a_classloader_3a_.gems.asciidoctor_minus_1_dot_5_dot_8.lib.asciidoctor.convert(uri:classloader:/gems/asciidoctor-1.5.8/lib/asciidoctor.rb:1521)

```

**Comment by ahus1** [comment]
From the stack trace I was able to derive a minimal example:

````
[asciimath]
++++
++++
````

**Comment by mojavelinux** [comment]
Yikes! This is a bug in core. Good thing this was caught before the 2.0.0 release ;) I'll get it fixed.

**Comment by ahus1** [comment]
Reported in https://github.com/asciidoctor/asciidoctor/issues/3118, fixed in asciidoctor 2.0.0.rc1. Waiting for next asciidoctorj release.

**Comment by ahus1** [comment]
This is fixed in Asciidoctor 2. Preview-Builds are available https://github.com/asciidoctor/asciidoctor-intellij-plugin/releases. A final release will be available after AsciidoctorJ is released as a final version.
